### DISCUSSION TITLE
Weird behavior while pasting links on existing Markdown links

### DISCUSSION THREAD

**Discussion by marc2332** [community/community#18422] [Projects and Issues]
Pasting normal links in some parts of Markdown links will cause them to generate weird Markdown links.

## Examples:

- Having `[blablabla](https://google.com)`, copy `https://google.com` and paste while selecting `blablabla`, result:
  -  `[[blablabla](https://google.com)](https://google.com)`

- Having `[blobloblo](https://google.com)`, copy `https://google.com` and paste while selecting `(https://google.com)`, result:
  -  `[blablabla][(https://google.com)](https://google.com)`
  
Tested in Firefox and Edge-_ium_.

I guess this affects all the other places where the Markdown editor is also used, besides Issues.

**Comment by kidonng** [comment]
I believe this is a recent regression since I remember it used to do the right thing.

EDIT: now I gave it a second thought, I think it's just that GitHub stop handling these special cases. Instead, they suggest [pasting without format using alternative shortcuts](https://github.blog/changelog/2022-05-19-updates-to-markdown-pasting-on-github/).
